pages/faq.md

```diff
@@ -77,10 +77,11 @@ Tap-and-hold on the Symbol selector button. Context menu with hide and protect o
 
 ### What are the most notable differences to OCAD?
 
-* Whole objects can be moved by dragging the dashed magenta bounding box.
-* There is a single tool for straight lines and Bézier curves.
-* Addition and removal of path points (vertices) can be achieved via **Edit** tool using various combinations of keyboard modifiers. See the status line at the bottom edge of the window for dynamically updated help. See also [Drawing](#drawing).
-* Dash points can be added and removed using the **Edit** tool after pressing **Ctrl-Space** on keyboard.
+* Whole objects can be moved by dragging the dashed magenta bounding box when [**Edit** tool](toolbars.md#tool_edit_point) is selected.

```

Comment:
I noticed that the tool is called "Edit objects" in the menu and in its documentation section. Do you think we should adjust the terminology in this file, too?

Reply:
Yes! Referring to the tools with their whole name makes a lot of sense to me.